### PR TITLE
Added cross-framework compatibility

### PR DESCRIPTION
**Pull request by axunonb**
Rewrote PR on top of latest merge #https://github.com/YAXLib/YAXLib/commit/5a566a121c1d9f86b379c15f76431914ab8f6e6a

### CODE REVIEW

**Comment by 304NotModified** [comment]
I'm afraid I'm not the right person to review this pr. 

It has been some years I used this library.

File: YAXLib/YAXLib.csproj [hunk 1]
```diff
@@ -19,6 +19,7 @@
     </ItemGroup>
     
     <ItemGroup>
+        <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.6" />
```
Comment (NKnusperer) [inline YAXLib/YAXLib.csproj:22]: For what is this needed? I've removed this in #122.

Comment (axunonb) [reply YAXLib/YAXLib.csproj:22]: Only required in case support for net46 should be supported.

**Comment by NKnusperer** [comment]
lgtm

**Comment by 304NotModified** [comment]
> lgtm

@NKnusperer please set a approval here:


image

**Review by NKnusperer — APPROVED** [review]

**Pull request merged**